src/main/webapp/js/user-page-loader.js

```diff
@@ -30,21 +30,23 @@ function setPageTitle() {
 }
 
 /**
- * Shows the message form if the user is logged in and viewing their own page.
+ * Shows the message form if the user is logged in and viewing every user page.
  */
-function showMessageFormIfViewingSelf() {
+function showMessageFormIfLoggedIn() {
   fetch('/login-status')
       .then((response) => {
         return response.json();
       })
       .then((loginStatus) => {
-        if (loginStatus.isLoggedIn &&
-            loginStatus.username == parameterUsername) {
+        if (loginStatus.isLoggedIn) {
           const messageForm = document.getElementById('message-form');
+        messageForm.action = '/messages? recipient=' + parameterUsername;
```

Comment:
I don't think there should be a space within '/messages?recipient='

Reply:
Updated the change in a new commit. Thanks for letting me know 😄